CMakeLists.txt

```diff
@@ -1,9 +1,186 @@
 cmake_minimum_required(VERSION 3.10.0)
+
+option (VERONA_DOWNLOAD_LLVM "Download cached version of LLVM" On)
+
+# This is used to create options that are passed through to the main Verona
+# build
+set (VERONA_EXTRA_CMAKE_ARGS)
+macro(verona_option variable message default)
+  option(variable message default)
+  list (APPEND VERONA_EXTRA_CMAKE_ARGS
+    "-D${variable}=${${variable}}"
+  )
+endmacro()
+
+verona_option(ENABLE_ASSERTS "Enable asserts even in release builds" OFF)
+verona_option(RT_TESTS "Including unit tests for the runtime" OFF)
+verona_option(USE_SCHED_STATS "Track scheduler stats" OFF)
+verona_option(USE_ASAN "Use address sanitizer" OFF)
+verona_option(VERONA_CI_BUILD "Disable features not sensible for CI" OFF)
+verona_option(USE_SYSTEMATIC_TESTING "Enable systematic testing in the runtime" OFF)
+verona_option(VERONA_EXPENSIVE_SYSTEMATIC_TESTING "Increase the range of seeds covered by systematic testing" OFF)
+verona_option(USE_CRASH_LOGGING "Enable crash logging in the runtime" OFF)
```

Comment:
Should these options include the `USE_MEASURE`, `USE_SCHED_STATS`, and `USE_QUICK_EXIT` options removed from the rt cmake?

Reply:
USE_SCHED_STATS has remained.  The others should have been removed already, as they don't control anything in Verona anymore.